src/runtime_src/core/tools/xbutil2/OO_MemWrite.cpp

```diff
@@ -142,13 +180,26 @@ XBU::verbose("SubCommand option: read mem");
   XBU::verbose(boost::str(boost::format("Device: %s") % xrt_core::query::pcie_bdf::to_string(xrt_core::device_query<xrt_core::query::pcie_bdf>(device))));
   XBU::verbose(boost::str(boost::format("Address: %s") % addr));
   XBU::verbose(boost::str(boost::format("Size: %s") % size));
+  XBU::verbose(boost::str(boost::format("Block count: %s") % count));
   XBU::verbose(boost::str(boost::format("Fill pattern: %s") % m_fill));
+  XBU::verbose(boost::str(boost::format("Input File: %s") % m_inputFile));
```

Comment:
It would be helpfult to indicate how many bytes of the given file will be written.